javascript/modules/client/src/internal/interfaces/core.ts

```diff
@@ -32,10 +33,12 @@ export interface IClientIpfsCore {
 }
 export interface IClientGraphQLCore {
   // Add here
+  getClient: () => GraphQLClient
```

Comment:
I didn't think that this could be a problem since it only had one 1 url, but since the subgraph hosted service is being deprecated, I don't know if we will deploy multiple endpoints so i'll add the functionality